cmd/hashsum/hashsum.go

```diff
@@ -55,6 +59,32 @@ func GetHashsumOutput(filename string) (out *os.File, close func(), err error) {
 	return out, close, nil
 }
 
+// HashStream produces hash from a stream
+func HashStream(ht hash.Type, outputBase64 bool, in io.ReadCloser) error {
+	hasher, err := hash.NewMultiHasherTypes(hash.NewHashSet(ht))
+	if err != nil {
+		return errors.Wrap(err, "Hash unsupported")
```

Comment:
According to Go style rules, error messages should start with a small letter.

This is because they get chained "error a: error b: error c" then eventually logged "Something went wrong: error a: error b: error c"

Reply:
I can change that. I was aware of it, but the code I copied from (existing hash functionality) had these exact messages, and also other errors within the same file used uppercase, so I chose to keep it. In general, this is something I've been thinking about several times, as there is a good mix throughout the code base. But sure, at least new code can do the right thing! :)